src/components/Section/shared/SubsectionConnector.jsx

```diff
@@ -0,0 +1,263 @@
+import React from 'react'
```

Comment:
This file got very verbose since it's combining all of the selectors for different section/question requirements based on form type. However it is my hope that with all of this data in the same place, we can suss out a configuration-based way to structure this data.